madgraph.sh

```diff
@@ -5,12 +5,25 @@ source: https://github.com/alisw/MadGraph
 requires:
  - "Python:slc.*"
  - "Python-system:(?!slc.*)"
+ - fastjet
+ - GoSam
+ - lhapdf
 build_requires:
   - alibuild-recipe-tools
 ---
 #!/bin/bash -e
 
-rsync -a --exclude='**/.git' --delete --delete-excluded $SOURCEDIR/ $INSTALLROOT/
+rsync -a --delete --exclude '**/.git' --delete-excluded $SOURCEDIR/ ./
+
+echo "generate e+ e- > u u~" >> test.mg
```

Comment:
Hello @mfasDa,

 I am not sure to guess what you are after with these transient lines, you echo sthg into a file and run madgraph for what seems a local test. You want to have a trace somewhere in the build logs ?

You may comment in the bash recipe about your intent.

Reply:
See recommendation in https://herwig.hepforge.org/tutorials/installation/prerequisite.html (section MadGraph)